### PR TITLE
fix(file component): Incorrect arguments were passed to function

### PR DESCRIPTION
**Pull request by kartik0709**
Incorrect arguments were passed to function

### CODE REVIEW

File: src/components/Inputs/File/File.js [hunk 1]
```diff
@@ -413,8 +412,9 @@ File.defaultProps = {
   baseWidth: 290,
   className: "",
   prefixClassName: "",
-  type: "",
-  dimensions: "",
+  type: ".png, .jpg",
+  dimensions: "450px X 450px",
```
Comment (Shivang-Bhandari) [inline src/components/Inputs/File/File.js]: can you comment and drop a source from where these standards were picked?

Comment (Shivang-Bhandari) [reply src/components/Inputs/File/File.js]: helps us keep a track of why these values were set as default in the first place

Comment (kartik0709) [reply src/components/Inputs/File/File.js]: 1. Media files are a default in Mogambo thats why I've picked that.
2. Wordpress default file size limit is 5MB.

Comment (kartik0709) [reply src/components/Inputs/File/File.js]: Given that these are just default Props I'm not sure if we including the source is that important.

**Pull request merged**